CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 - Enh #801: Deprecate `AbstractSchema::normalizeRowKeyCase()` method (@Tigrov)
 - Enh #801: Deprecate `SchemaInterface::getRawTableName()` and add `QuoterInterface::getRawTableName()` method (@Tigrov)
 - Enh #801: Deprecate `SchemaInterface::isReadQuery()` and add `DbStringHelper::isReadQuery()` method (@Tigrov)
+- Enh #801: Remove unnecessary symbol `\\` from `rtrim()` function inside `DbStringHelper::baseName()` method (@Tigrov)
```

Comment:
```suggestion
- Bug #801: Remove unnecessary symbol `\\` from `rtrim()` function inside `DbStringHelper::baseName()` method (@Tigrov)
```

Reply:
I think this is not a bug, just removed unnecessary symbol.

Reply:
phpdoc:

```
Returns the trailing name part of a path.

This method is similar to the php function `basename()` except that it will treat both \ and / as directory
separators, independent of the operating system.
```
     
Before remove only `/`, but not `\`. This change fixes it. I think, it is bug.